content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md

```diff
@@ -113,4 +113,5 @@ spec:
 - Huge page usage in a namespace is controllable via ResourceQuota similar
   to other compute resources like `cpu` or `memory` using the `hugepages-<size>`
   token.
+- EmptyDir Volume/VolumeMount usage is not required if the pod consumes huge pages with `shmat`/`shmget` system calls or `mmap` with the `MAP_HUGETLB` flag.
```

Comment:
```suggestion
- EmptyDir volumes or volume mounts are not required if a Pod consumes huge pages
  with `shmat`/`shmget` system calls or `mmap` with the `MAP_HUGETLB` flag.
- Huge page usage in a namespace is controllable via ResourceQuota similar
  to other compute resources like `cpu` or `memory` using the `hugepages-<size>`
  token.
```

Reply:
I think this is out of scope for this pull request.

Reply:
Not a strong opinion. I was expecting that two items related to `shmat/shmget` calls sit next to each other.

Reply:
The text that @tengqm suggested makes more sense to me; for
```markdown
- EmptyDir Volume/VolumeMount usage is not required if the pod consumes huge
  pages with `shmat`/`shmget` system calls or `mmap` with the `MAP_HUGETLB` flag.
```
I didn't really know what was meant without more context.

Here's my tweaked version. @kirinrastogi I recommend you do accept either my suggestion
or @tengqm's.
```suggestion
- `emptyDir` volumes or volume mounts are not required if a Pod consumes huge pages
  with `shmat`/`shmget` system calls, or by using `mmap` with the `MAP_HUGETLB` flag.
- Huge page usage in a namespace is controllable via ResourceQuota similar
  to other compute resources like `cpu` or `memory`. You define requests and limits for
  hugepages of different sizes by using the `hugepages-<size>` token. For example,
  `hugepages-2Mi` is the token for the total amount of 2MiB pages that a container or Pod
  could use.
```


Optionally (not required for _this_ PR): change other instances of EmptyDir to `emptyDir`, which is the capitalization we use in the API (eg, in a JSON serialization).